pkg/storage/get_access_token.go

```diff
@@ -0,0 +1,37 @@
+package storage
+
+import (
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/common-fate/ddb"
+	"github.com/common-fate/granted-approvals/pkg/access"
+	"github.com/common-fate/granted-approvals/pkg/storage/keys"
+)
+
+type GetAccessToken struct {
+	ID     string
```

Comment:
can be more specific here

```suggestion
	RequestID     string
```